### PR TITLE
feat(image-uploader-delete): add functionality to delete files

### PR DESCRIPTION
**Pull request by jwinding**
## Explain the changes you’ve made

Adds functionality to use the attachments-delete endpoint to remove uploaded files from the ImageDisplay component. 

## Explain why these changes are made

If you upload the wrong image by accident, it's obviously good if you can delete it. 

## Explain your solution

Implements a small helper-function to send delete requests, and then uses it in the ImageDisplay component to delete an uploaded file when the user removes the file from the list. Also fixes a typo in the ImageUploader, uploadedFilename->  uploadedFileName.

## How to test the changes?

Checkout the branch, and go into a form that has an ImageUploader. Then open the S3 bucket (called attachments-s3-{some id}) in your AWS console. Then try uploading a file, and check that it appears in the S3. Then, remove it by pressing the x button, and again check that it is removed from the S3 bucket. 

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.
- [x] Building the Application on a Android device/simulator.

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-cyygz6 Bilduppladdningskomponenten ska kunna ta bort uppladdade filer](https://app.clickup.com/t/cyygz6)

**Review by jonatanhanson — CHANGES_REQUESTED** [review]
Functionality works great. Have left a comment with a small fix.

File: source/helpers/FileUpload.ts [hunk 1]
```diff
@@ -64,3 +64,28 @@ export const uploadFile = async ({
...
+  endpoint: string;
+  fileName: string;
+}
+export const deleteUploadedFile = async ({ endpoint, fileName }: FileDeleteParams) => {
+  const requestUrl = await buildServiceUrl(`${endpoint}/${fileName}`);
+  const token = await StorageService.getData(ACCESS_TOKEN_KEY);
+  const bearer = token || '';
+
+  try {
+    await axios({
```
[the remark this answers is not in the document]
Comment (jwinding) [reply source/helpers/FileUpload.ts:78]: Yeah, for sure. Actually this whole helper-function is pretty much duplicating the existing remove helper, so I can just remove it alltogether. I'll do that.

**Review by jonatanhanson — APPROVED** [review]
Awesome!

**Pull request merged**